main.go

```diff
@@ -116,6 +126,21 @@ func snapshot() error {
 	return nil
 }
 
+func getLifetimeStaleDays() (int, error) {
+	const defaultLifetimeStaleDays = 14
+	lifetimeStaleDays := os.Getenv("LIFETIME_STALE_DAYS")
+	if len(lifetimeStaleDays) == 0 {
+		return defaultLifetimeStaleDays, nil
+	}
+
+	integerLifetimeStaleDays, err := strconv.Atoi(lifetimeStaleDays)
+	if err != nil {
+		return 0, fmt.Errorf("failed to read Datadog Config: %w", err)
```

Comment:
maybe it should be "failed to convert lifetimeStaleDays"

Reply:
oops, thank you!